docs/navigation-prop.md

```diff
@@ -6,25 +6,23 @@ sidebar_label: Navigation prop
 
 Each `screen` component in your app is provided with the `navigation` prop automatically. It looks like this:
 
-* `this.props.navigation`
-  * `navigate` - go to another screen, figures out the action it needs to take to do it
```

Comment:
please keep the current description with "figures out the action it needs to take to do it"

Reply:
First - much thanks for all of the feedback.

Could we update the wording? I'm not sure what it means currently. Looking at the code, it seems like `navigate` just dispatches a navigation action, but I didn't follow it any further.

Reply:
What this is trying to say is that 

* when you're on a stack and need to go from A to B, `navigate(B)` will push B onto the stack (in stack navigator, the behavior also differs based on whether or not a route is on the stack already - this will be covered in another docs page that does not exist yet).
* when you're in a tab navigator and navigate to another tab, the `navigate` call gets you there.
* when you're in a drawer and need to go to another route in the drawer...

If you find a better way to say this, feel free to add it to the PR.